permissions/component-bitbucket-kubernetes-credentials-plugin.yml

```
@@ -0,0 +1,7 @@
+---
+name: "bitbucket-kubernetes-credentials-plugin"
+github: "jenkinsci/bitbucket-kubernetes-credentials-plugin"
+paths:
+- "org/jenkins-ci/bitbucket-kubernetes-credentials-plugin"
```

```suggestion
- "io/jenkins/plugin/bitbucket-kubernetes-credentials"
```

This is the `groupId` (https://github.com/jenkinsci/bitbucket-kubernetes-credentials-plugin/blob/master/pom.xml#L11) + `artifactId` (https://github.com/jenkinsci/bitbucket-kubernetes-credentials-plugin/blob/master/pom.xml#L12) in path form (replace . with /)